plugin/al/mayautils/AL/maya/utils/PluginTranslatorOptions.cpp

```diff
@@ -258,7 +258,7 @@ void PluginTranslatorOptionsInstance::parse(MString key, const MString& value)
             if (opt->name == key) {
                 auto& optData = set.m_options[i];
                 switch (opt->type) {
-                case OptionType::kBool: optData.m_bool = value.asInt(); break;
+                case OptionType::kBool: optData.m_bool = (value.asInt() > 0); break;
```

Comment:
I wonder why other compilers than VS2015 don't report this warning (which then generates errors since we have warnings as errors enabled)